src/app/src/pages/eval/components/ResultsView.tsx

```diff
@@ -518,9 +518,14 @@ export default function ResultsView({
         sx={{
           display: 'flex',
           flexDirection: 'column',
-          // TODO(Will): This is a hack because the flexbox must have a fixed height in order for the pagination footer to be
-          // stuck to the bottom of the viewport; ideally the parent nodes are flexboxes.
-          height: `calc(100vh - 81px)`,
+        }}
+        ref={(el: HTMLDivElement | null) => {
+          const top = el?.getBoundingClientRect().top;
+          if (top) {
+            // TODO(Will): This is a hack because the flexbox must have a fixed height in order for the pagination footer to be
+            // stuck to the bottom of the viewport; ideally the parent nodes are flexboxes.
```

Comment:
Is that really true? Can you not do `position: absolute` or `position: sticky` to achieve something similar?

Reply:
Otherwise the container will be as tall as its content